### PR TITLE
Remove images metadata

### PR DESCRIPTION
**Pull request by mikeger**
# Issue

For privacy reasons it is necessary to remove the various information from the images that are being transferred via Wire.

### CODE REVIEW

File: Sources/Image Processing/NSData+MediaMetadata.swift [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        swiftMetadataProperties[String(kCGImagePropertyMakerOlympusDictionary)] = kCFNull
+        swiftMetadataProperties[String(kCGImagePropertyMakerPentaxDictionary)] = kCFNull
+        
+        return swiftMetadataProperties as CFDictionary
+    }()
+    
+    
+    // Removes the privacy-related metadata tags from the binary image (see nullMetadataProperties).
+    // Supports JPEG, TIFF, PNG and other image (container) formats/types.
+    // @throws MetadataRemovalError
```
Comment (typfel) [inline Sources/Image Processing/NSData+MediaMetadata.swift]: This does't match name of error thrown `MetadataError `

File: Sources/Image Processing/NSData+MediaMetadata.swift [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        swiftMetadataProperties[String(kCGImagePropertyMakerPentaxDictionary)] = kCFNull
+        
+        return swiftMetadataProperties as CFDictionary
+    }()
+    
+    
+    // Removes the privacy-related metadata tags from the binary image (see nullMetadataProperties).
+    // Supports JPEG, TIFF, PNG and other image (container) formats/types.
+    // @throws MetadataError
+    @objc public func wr_removingImageMetadata() throws -> NSData {
```
Comment (typfel) [inline Sources/Image Processing/NSData+MediaMetadata.swift]: The name sticks out a bit with the verb in present tense. How about `wr_imageDataWithoutMetadata`?

Comment (mikeger) [inline Sources/Image Processing/NSData+MediaMetadata.swift]: It's according to the swift naming guideline, mutator should be `remove...`, producing goes in present tense:
```
When the operation is naturally described by a verb, use the verb’s imperative for the mutating method and apply the “ed” or “ing” suffix to name its nonmutating counterpart.
```

Comment (mikeger) [inline Sources/Image Processing/NSData+MediaMetadata.swift]: I guess as it is used from Objective-C right now, would make sense to keep swift naming in `Data` and use the other one in Obj-C.

**Comment by mikeger** [comment]
@typfel updated the API naming

**Pull request merged**